azure-pipelines.yml

```diff
@@ -36,13 +36,14 @@ jobs:
       keyVaultName: gadgetron-build-secrets
   - script: |
         set -euo pipefail
+        echo ${{ secrets.CR_PAT }} | docker login ghcr.io -u gadgetron --password-stdin
         curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
           && curl -s -L https://nvidia.github.io/libnvidia-container/stable/deb/nvidia-container-toolkit.list | sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list \
           && sudo apt-get update
         sudo apt-get install -y nvidia-container-toolkit
         sudo nvidia-ctk runtime configure --runtime=docker
         sudo systemctl restart docker
-        docker run --rm --gpus all nvidia/cuda:12.3.0-base-ubuntu20.04 nvidia-smi
+        docker run --rm --gpus all ghcr.io/gadgetron/gadgetron/cuda:12.3.0-base-ubuntu22.04
```

Comment:
I think you need to run `nvidia-smi` to actually test whether it works. I made that image public.

Reply:
Seems so. otherwise, it needs the login to github container  registry.

Reply:
fixed